gnat2goto/driver/tree_walk.adb

```diff
@@ -154,18 +154,28 @@ package body Tree_Walk is
    with Pre  => Nkind (N) in N_Op,
         Post => Kind (Do_Operator_Simple'Result) in Class_Expr;
 
-   function Do_Operator_Mod (LHS : Irep; Op_Kind : Irep_Kind;
-                                    RHS : Irep; Ret_Type : Irep)
-                                    return Irep
+   function Do_Operator_Mod
+     (LHS : Irep;
+      Op_Kind : Irep_Kind;
+      RHS : Irep; Ret_Type : Irep)
```

Comment:
Argh :-( Since we are doing it, can't we just get one parameter per line? The inconsistency here is killing me. Sorry to be a pain in the rear...

Reply:
Also this seems to be just a declaration? Why not move it to `tree_walk.ads` while we're at it?

Reply:
@NlightNFotis `tree_walk.ads` should only contain functions that should be exported to people importing `tree_walk`. It already contains way too much stuff IMHO